decko/lib/decko/application.rb

```diff
@@ -21,7 +21,7 @@ class Application < Rails::Application
     end
 
     initializer :load_card, after: :load_config_initializers, group: :all do
-       Card
+      Card if Cardio.load_card?
```

Comment:
This was causing problems (newly exposed by the decko command line exit handling) in calls processed before the database is created.